src/main/java/vg/civcraft/mc/civmodcore/CivModCorePlugin.java

```diff
@@ -1,28 +1,52 @@
 package vg.civcraft.mc.civmodcore;
 
+import org.bukkit.configuration.serialization.ConfigurationSerialization;
+import org.bukkit.event.HandlerList;
+import vg.civcraft.mc.civmodcore.api.ItemAPI;
+import vg.civcraft.mc.civmodcore.chatDialog.ChatListener;
+import vg.civcraft.mc.civmodcore.chatDialog.DialogManager;
+import vg.civcraft.mc.civmodcore.dao.ManagedDatasource;
+import vg.civcraft.mc.civmodcore.inventorygui.ClickableInventoryListener;
+import vg.civcraft.mc.civmodcore.itemHandling.NiceNames;
 import vg.civcraft.mc.civmodcore.playersettings.gui.ConfigCommand;
+import vg.civcraft.mc.civmodcore.scoreboard.ScoreBoardListener;
 
 public class CivModCorePlugin extends ACivMod {
 
 	private static CivModCorePlugin instance;
 
 	@Override
-	public void onEnable() {
+	public synchronized void onEnable() {
 		super.onEnable();
-		// needed for some of the apis
 		instance = this;
-		// we cant use the command annotation thing here, because the annotation
```

Comment:
This comment should stay for the next person trying to fix this

Reply:
Fair enough. I thought the comment specifying it must be done traditionally would be enough but if not, this is so minor this comment isn't really worth it :P